pkg/cluster/machine.go

```diff
@@ -240,5 +240,5 @@ func (m *Machine) IsIgnite() (b bool) {
 		igniteChecked = true
 	}
 
-	return
+	return b
```

Comment:
The statement before means the same thing, since `b` is specified as a named return value.
In other words, you did not make any change to the meaning of the code.